src/main/fc/rc_modes.h

```diff
@@ -56,6 +56,7 @@ typedef enum {
     BOXCAMERA1      = 29,
     BOXCAMERA2      = 30,
     BOXCAMERA3      = 31,
+    BOXLIGHTS       = 32,
```

Comment:
There already is `BOXLLIGHTS      = 13,`

Reply:
I added a mode in case somebody wants to use RGB and standard lighting at the same time but I can modify the code to use the LLIGHTS mode without a problem.

Reply:
LLIGHTS are not used, it's not connected with RGB LEDs at low. You mistake it with `BOXLEDLOW`

Reply:
`BOXLLIGHTS` stands for Landing Lights

Reply:
Done. Now using old unused BOXLLIGHTS IDs.